lib/adapters/test-result/playwright.ts

```diff
@@ -159,7 +159,7 @@ const getHistory = (steps?: PlaywrightTestResult['steps']): TestStepCompressed[]
         [TestStepKey.Name]: step.title,
         [TestStepKey.Args]: [],
         [TestStepKey.IsFailed]: Boolean(step.error),
-        [TestStepKey.TimeStart]: step.startTime.getTime(),
+        [TestStepKey.TimeStart]: step.startTime instanceof Date ? step.startTime.getTime() : new Date(step.startTime).getTime(),
```

Comment:
When launched via "npx playwright test", its intance of Date.
When launched via "npx html-reporter --tool playwright", its ISO string